### ISSUE TITLE
add the following badge to the readme.md

### ISSUE THREAD

**Issue by MadushaS** [UOKCS-19/UOKCS-19.github.io#17]
```markdown
[![DeepSource](https://deepsource.io/gh/UOKCS-19/UOKCS-19.github.io.svg/?label=active+issues&token=G05jCYgHjy_6OHj6OVgxb6LO)](https://deepsource.io/gh/UOKCS-19/UOKCS-19.github.io/?ref=repository-badge)
```

**Comment by the-amazing-atharva** [comment]
Hi!
I would like to contribute to this issue!

**Comment by MadushaS** [comment]
The issue is now resolved in pull #19
